server/helpers/middleware.js

```diff
@@ -59,3 +60,18 @@ export const isAdmin = async (req, res, next) => {
   }
   return null;
 };
+
+export const paramIdIsValid = async (req, res, next) => {
+  const { id } = req.params;
+  const { ObjectId } = mongoose.Types;
+
+  if (ObjectId.isValid(id)) {
+    next();
+  } else {
+    return res.status(httpStatus.PRECONDITION_FAILED).json({
+      success: false,
+      message: 'ID should be a valid mongo object ID',
```

Comment:
We don't want to reveal the technology we are using to everyone consuming the API.

```suggestion
      message: 'Invalid Id supplied',
```